beater/beatertest/config.go

```diff
@@ -0,0 +1,21 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package beatertest
+
+// MockBeatVersion returns a beat version for testing
+func MockBeatVersion() string { return "8.0.0" }
```

Comment:
I can see where it is used, but I can't grasp how it happened that we need this now...

Reply:
So far we used e.g. `defaultConfig("7.0.0")` all over the place in testing. I think it is nicer to introduce a dedicated mock version that can be used everywhere. This also allows for easier upgrade the version in the future.